packages/compiler/core/checker.ts

```diff
@@ -2265,15 +2266,19 @@ export function createChecker(program: Program): Checker {
       return;
     }
 
-    if (inheritedPropertyNames?.has(newProp.name)) {
-      program.reportDiagnostic(
-        createDiagnostic({
-          code: "override-property",
-          format: { propName: newProp.name },
-          target: diagnosticTarget ?? newProp,
-        })
-      );
-
+    if (inheritedProperties?.has(newProp.name)) {
+      const inheritedProp = inheritedProperties.get(newProp.name)!;
+      const [isAssignable, _] = isTypeAssignableTo(newProp.type, inheritedProp.type, newProp);
```

Comment:
not part of this PR but I don't like "assignable" since our specs I think only care about subtyping, we don't have a notion of assignability yet.

Reply:
From what I remember this was on how the typescript checker did the type relation it had a `isTypeRelatedTo` function and you'd pass the relation type(inheritence, assignment, etc.) not sure what other there was but assignment felt like the correct description